languages/nl.lang

```diff
@@ -319,7 +318,7 @@ $PALANG['pAdminEdit_domain_quota'] = 'Domein Quota';
 $PALANG['transport'] = 'Transport';
 $PALANG['backupmx'] = 'Backup-MX';
 $PALANG['pAdminEdit_domain_transport_text'] = 'Definieer transport';
-$PALANG['pAdminEdit_domain_backupmx'] = 'Definieer of deze Mailserver een Backup MX is voor het domain';
+$PALANG['pAdminEdit_domain_backupmx'] = 'Mailserver is Backup MX';
```

Comment:
are you sure you want to revert this NL to EN ?

Reply:
yes, because the text is too long and does not cover the load well